worlds/hades/Items.py

```diff
@@ -0,0 +1,422 @@
+    "FountainElysiumItem": ItemData(hades_base_item_id+56, True),
+    "UrnsOfWealth1Item": ItemData(hades_base_item_id+57, True),
+    "UrnsOfWealth2Item": ItemData(hades_base_item_id+58, True),
+    "UrnsOfWealth3Item": ItemData(hades_base_item_id+59, True),
+    "InfernalTrove1Item": ItemData(hades_base_item_id+60, True),
+    "InfernalTrove2Item": ItemData(hades_base_item_id+61, True),
+    "InfernalTrove3Item": ItemData(hades_base_item_id+62, True),
+    "KeepsakeCollectionItem": ItemData(hades_base_item_id+63, True),
+    "DeluxeContractorDeskItem": ItemData(hades_base_item_id+64, True),
+    "VanquishersKeepItem": ItemData(hades_base_item_id+65, True),
+    "FishingRodItem": ItemData(hades_base_item_id+66, True),
+    "CourtMusicianSentenceItem": ItemData(hades_base_item_id+67, True),
+    "CourtMusicianStandItem": ItemData(hades_base_item_id+68, True),
+    "PitchBlackDarknessItem": ItemData(hades_base_item_id+69, True),
+    "FatedKeysItem": ItemData(hades_base_item_id+70, True),
+    "BrilliantGemstonesItem": ItemData(hades_base_item_id+71, True),
+    "VintageNectarItem": ItemData(hades_base_item_id+72, True),
+    "DarkerThirstItem": ItemData(hades_base_item_id+73, True),
+}
+
+item_table_hidden_aspects : Dict[str, ItemData] ={
+    "SwordHiddenAspect" : ItemData(hades_base_item_id+74, True),
+	"BowHiddenAspect": ItemData(hades_base_item_id+75, True),
+	"SpearHiddenAspect": ItemData(hades_base_item_id+76, True),
+	"ShieldHiddenAspect": ItemData(hades_base_item_id+77, True),
+	"FistHiddenAspect": ItemData(hades_base_item_id+78, True),
+	"GunHiddenAspect": ItemData(hades_base_item_id+79, True)
+}
+
+item_table_traps : Dict[str, ItemData] ={
+    "MoneyPunishment" : ItemData(hades_base_item_id+80, False, False, True),
+	"HealthPunishment": ItemData(hades_base_item_id+81, False, False, True),
+}
+
+item_table_helpers : Dict[str, ItemData] ={
+    "MaxHealthHelper" : ItemData(hades_base_item_id+82, False, False, False),
+    "BoonBoostHelper" : ItemData(hades_base_item_id+83, False, False, False),
+    "InitialMoneyHelper" : ItemData(hades_base_item_id+84, False, False, False),
+}
+
+def create_filler_pool_options(options):
+    item_filler_options = []
+    if (options.darkness_pack_value.value > 0):
+        item_filler_options.append("Darkness")
+    if (options.keys_pack_value.value > 0):
+        item_filler_options.append("Keys")
+    if (options.gemstones_pack_value.value > 0):
+        item_filler_options.append("Gemstones")
+    if (options.diamonds_pack_value.value > 0):
+        item_filler_options.append("Diamonds")
+    if (options.titan_blood_pack_value.value > 0):
+        item_filler_options.append("TitanBlood")
+    if (options.nectar_pack_value.value > 0):
+        item_filler_options.append("Nectar")
+    if (options.ambrosia_pack_value.value > 0):
+        item_filler_options.append("Ambrosia")
+    if (len(item_filler_options)==0):
+        item_filler_options.append("Darkness")
```

Comment:
```suggestion
    if not item_filler_options:
        item_filler_options.append("Darkness")
```
`if not` should be True if the list is empty